### DISCUSSION TITLE
can/should i use spring modulith without spring

### DISCUSSION THREAD

**Discussion by ddeboer81** [spring-projects/spring-modulith#606] [Q&A]
i have a Jakarta EE project where i would like to use spring modulith.
Does it work well without spring ?

Do i need a workaround ?

**Comment by breun** [comment]
The various Spring Modulith artifacts have all kinds of dependencies on various Spring artifacts, so depending on your definition of 'without Spring' I would say that you are going to be using Spring when you want to use Spring Modulith.
